ConsoleApp2/Snake.cs

```diff
@@ -42,24 +48,50 @@ public void Turn()
                 way = "up";
                 PositionY -= speed;
             }
-            if (key.Key == ConsoleKey.S && way != "up" && PositionY < SizeY)
+            if (key.Key == ConsoleKey.S && way != "up" && PositionY < sizeY)
             {
                 way = "down";
                 PositionY += speed;
             }
         }
 
-        /*public bool CheckTouch(Object obj)
+        public bool CheckTouch(Object obj)
         {
             if (obj is Snake)
             {
-                Snake temp;
-                temp = (Snake)obj;
-                if (PositionX == temp.PositionX && PositionY == temp.PositionY)
+                Snake snake;
+                snake = (Snake)obj;
+                if (PositionX == snake.PositionX && PositionY == snake.PositionY)
                 {
-
+                    return true;
+                }
+                else { return false; }
```

Comment:
"else" statement is excess code

For ex. we can write code that make same sence
if (...)
{
return true;
}
return false;

Reply:
more short version
if (...)
      return true;

return false;